faker/options.go

```diff
@@ -0,0 +1,23 @@
+package faker
+
+type Option func(*faker)
+
+func WithMaxDepth(depth int) Option {
+	return func(f *faker) {
+		f.maxDepth = depth
+	}
+}
+
+func WithSkipEFace() Option {
+	return func(f *faker) {
+		f.ignoreEFace = true
+	}
+}
+
+func WithSkipFields(fieldName ...string) Option {
+	return func(f *faker) {
+		for _, field := range fieldName {
+			f.skipFields[field] = struct{}{}
```

Comment:
Can you give an example of fields we skip? I remember needing to skip only fields faker couldn't fake for some reason (usually a bug in faker). Maybe we should just ignore those

Reply:
> Can you give an example of fields we skip? I remember needing to skip fields only when faker couldn't fake them for some reason (usually a bug in faker). Maybe we should just ignore those

Just search for `faker.FakeDataSkipFields`

Reply:
What happens if we use our faker and don't use `FakeDataSkipFields`?
I remember needing to skip fields due to infinite recursion, but that shouldn't happen anymore.

Reply:
I'll give it a try

Reply:
@erezrokah seems to work (I wasn't able to test wafv2 and iot, but others are OK) so removed the option in [`006b015` (#287)](https://github.com/cloudquery/plugin-sdk/pull/287/commits/006b0158290d1ae019b382a3114d54a21fe05fdd)